networking/aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="aws-load-balancer"]
+= AWS Load Balancer Operator
+include::_attributes/common-attributes.adoc[]
+:context: aws-load-balancer-operator
+
+toc::[]
+
+
+The AWS Load Balancer Operator (ALBO) is a Red-Hat supported Operator that users can optionally install on SRE-managed
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+clusters.  The ALBO  manages the life cycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in
```

Comment:
```suggestion
clusters.  The ALBO manages the lifecycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in
```
Per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l), `lifecycle` is one word.